src/osd/osd_types.h

```diff
@@ -5578,6 +5582,38 @@ struct object_manifest_t {
     chunk_map.clear();
   }
 
+  /**
+   * calc_refs_to_inc_on_set
+   *
+   * Takes a manifest and returns the set of refs to
+   * increment upon set-chunk
+   *
+   * l should be nullptr if there are no clones, or 
+   * l should be a manifest for previous clone 
+   *
+   */
+  void calc_refs_to_inc_on_set(
+    const object_manifest_t* g, ///< [in] manifest for clone > *this
+    const object_manifest_t* l, ///< [in] manifest for clone < *this
+    object_ref_delta_t &delta    ///< [out] set of refs to drop
+  ) const;
+
+  /**
+   * calc_refs_to_drop_on_modify
+   *
+   * Takes a manifest and returns the set of refs to
+   * drop upon modification 
+   *
+   * l should be nullptr if there are no clones, or 
+   * l should be a manifest for previous clone 
```

Comment:
Replace this bit with "l and g may each be null if the corresponding clone does not exist."

Reply:
Add "*this contains the set of new references to set"